### PR TITLE
Adding addscripts partial

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: templates/addscripts.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            s.parentNode.insertBefore(o, s.nextSibling);
+        }
+
+        if(cutsTheMustard){
+            addScript(polyfillService);
+            addScript(mainJs);
+        }
+
+
+    }('http://polyfill.webservices.ft.com/v1/polyfill.min.js?callback=FT.Next.init', '/{{__name}}/main.js'));
```
Comment (commuterjoy) [inline templates/addscripts.html]:
```
 [
  'http://polyfill.webservices.ft.com/v1/polyfill.min.js?callback=FT.Next.init', 
  '/{{__name}}/main.js'
 ]
```

File: templates/addscripts.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            document.documentElement.classList.add('js-success');
+            var o = document.createElement('script');
+            o.async = o.defer = true;
+            o.src = src;
+            var s = document.getElementsByTagName('script')[0];
+            s.parentNode.insertBefore(o, s.nextSibling);
+        }
+
+        if(cutsTheMustard){
+            addScript(polyfillService);
```
Comment (commuterjoy) [inline templates/addscripts.html]:
```
scripts.forEach(function (path) {
    addScript(path)
})
```

File: templates/addscripts.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<script>
+
+    (function (polyfillService, mainJs) {
```
Comment (commuterjoy) [inline templates/addscripts.html]: scripts

**Comment by commuterjoy** [comment]
:+1: but i'd pass an array of paths

File: templates/addscripts.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            o.src = src;
+            var s = document.getElementsByTagName('script')[0];
+            s.parentNode.insertBefore(o, s.nextSibling);
+        }
+
+        if(cutsTheMustard){
+            scripts.forEach(addScript);
+        }
+    }([
+        'http://polyfill.webservices.ft.com/v1/polyfill.min.js?callback=FT.Next.init',
```
Comment (matthew-andrews) [inline templates/addscripts.html]: not https compatible => `//polyfill.webservices.ft.com/v1/polyfill.min.js?callback=FT.Next.init`?

**Pull request merged**